src/Uno.Extensions.Navigation/NavigationRequestExtensions.cs

```diff
@@ -91,14 +91,6 @@ public static class NavigationRequestExtensions
 		}
 
 		var route = new Route(qualifier, routeBase, path, paras);
-		if ((route.IsBackOrCloseNavigation() && !route.IsClearBackstack()) &&
```

Comment:
This is no longer required since the conversion to Option is handled with the creation of NavigationResponse objects